GOVERNANCE.md

```diff
@@ -0,0 +1,205 @@
+
+There are different means and ways to contribute to Eiffel Community in order to be a contributor, such as issuing and reviewing pull requests, creating issues in project backlog, or taking part in technical discussions.
+Determination of the complete criteria for contributors is the responsibility of Eiffel Technical Committee with the input from Eiffel Community at large.
+
+Contributors are eligible to vote on Technical Committee elections. Contributors do not have the rights to accept pull requests.
+
+Being contributor is a prerequisite for becoming a maintainer to the projects they are contributing to.
+In order for a contributor to become a maintainer, the individual must be nominated by one of the existing maintainers with a reference to the individual's history of contributions to the project.
+The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
+
+### Maintainers
+
+Maintainers are active contributors and participants in the projects they are maintainers of. Maintainers have the rights to accept pull requests.
+
+Maintainers are eligible to vote on Technical Committee elections. They can be nominated as members to the Technical Committee during Technical Committee elections.
+
+Maintainers have right to nominate contributors to become maintainers and vote on the maintainer promotions for the projects they are maintainers of.
+
+Individuals can be maintainers to multiple projects.
+
+## Technical Committee
+
+The Technical Committee is responsible for the oversight of the Eiffel Open Source Project. Technical Committee is comprised of **X** voting members.
+
+The term for Technical Committee is **one year**.
+
+The responsibilities of the Technical Committee are
+
+* coordination of the technical direction of the project
+* discussions, seeking consensus, and where necessary, voting on technical matters that affect multiple projects
+* serving as point of contact with regards to licensing matters
+* determination and amendment of the size, makeup, and the procedure for Technical Committee
+* establishment of the project creation and archival processes, approving and archiving projects
+* creation and amendment of community governance, election, contribution, and code of conduct guidelines
+* enforcement and reminding of code of conduct processes
+* creation and amendment of the criteria for maintainer promotion and removal
+* creation or elimination of project and community roles
+* appointing community members to interact with other open source communities
+* coordinating event participation and communications regarding the project
+
+Eiffel Project aims to operate as a consensus based community, if any decision requires a vote to move the project forward, the members of the Technical Committee will vote on a one vote per member basis.
```

Comment:
I think we need to be clear on naming, are we referring to the eiffel protocol repo here or to the Eiffel-community and all of it's repos/projects. Again this might be me misunderstanding naming conventions and structure overall.

Reply:
I'll refer to them as "Eiffel Community Open Source Projects".